door43_tools/templaters.py

```diff
@@ -33,6 +39,11 @@ def run(self):
         self.rc = RC(self.source_dir)
         with open(self.template_file) as template_file:
             self.template_html = template_file.read()
+            soup = BeautifulSoup(self.template_html, 'html.parser')
+            soup.body['class'] = soup.body.get('class', []) + [self.resource_type]
```

Comment:
@phillip-hopper Ok, found I need to be making it an array, and that this is how people add a class when they don't know if it exists or not.

Also realized that 'ulb' and 'udb' were putting that as their class, butthe project-page.css only handles 'bible' so added 'bible' but didn't want to put the whole list of Bible book types, as we already do three other places, so I made the array a constant...wasn't sure best place to put it other than usfm2html_converter.